src/styles/mobile-styles.css

```diff
@@ -0,0 +1,80 @@
+.nav-list-mobile {
+  overflow-y: scroll;
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  background-color: #191e1e;
+  z-index: 3;
+  padding: 40px 60px;
+  box-sizing: border-box;
+  transform: translateY(-100%);
+  transition: transform 0.35s cubic-bezier(0.694, 0, 0.335, 1);
+}
+
+.nav-list-mobile span,
```

Comment:
Please don't use element selectors and avoid nested selectors, plain css class would be almost always better